godot-macros/src/docs/markdown_converter.rs

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+//! Converts [markdown](https://en.wikipedia.org/wiki/Markdown) to [BBCode](https://en.wikipedia.org/wiki/BBCode).
+use markdown::mdast::Node;
+use markdown::{to_mdast, ParseOptions};
+use std::collections::HashMap;
+
+pub fn to_bbcode(md: &str) -> String {
+    // > to_mdast() never errors with normal markdown so feel free to unwrap
+    let n = to_mdast(md, &ParseOptions::gfm()).unwrap();
```

Comment:
```suggestion
    // to_mdast() never errors with normal Markdown, so unwrap is safe.
    let n = to_mdast(md, &ParseOptions::gfm()).unwrap();
```

Reply:
its a quote from the documentation though...

Reply:
We don't need to copy-paste mistakes, furthermore it isn't obvious that `>` means "original quote". The important part is to convey the meaning and stick to our comment style.